content-security-policy/reporting/report-only-in-meta.sub.html

```diff
@@ -33,6 +33,12 @@
       script.defer = true;
       script.src = '../support/checkReport.sub.js?reportExists=false'
       document.body.appendChild(script);
+
+      // Immediately declare a test so that the harness does not infer
+      // completion if the image loads before the script.
+      var checkReportTest = async_test("checkReport tests loaded");
+      script.onload = checkReportTest.step_func_done();
```

Comment:
Is there a chance of the script element onload event being fired before we set the handler? Could we move the document.body.appendChild(script); line after this?

Reply:
DOM event handlers do not run in parallel, but I'm happy to relocate if you feel it improves readability.

Reply:
I think it's nice to set all the properties on the element before adding it to the DOM but it does not really matter.